lib/middleware/apiHeader.js

```diff
@@ -1,17 +1,10 @@
-const { join } = require('path')
-const { URL } = require('url')
 const { Router } = require('express')
 
 function factory (api) {
   const router = new Router()
 
   router.use((req, res, next) => {
-    const docUrl = new URL(req.absoluteUrl())
-
-    docUrl.pathname = join(req.baseUrl, api.path)
-    docUrl.search = ''
-
-    res.setLink(docUrl.toString(), 'http://www.w3.org/ns/hydra/core#apiDocumentation')
+    res.setLink(api.term.value, 'http://www.w3.org/ns/hydra/core#apiDocumentation')
```

Comment:
I don't think it makes sense to resolve the API URL in two places. Hence, the term set directly as the link